setup.cfg

```diff
@@ -17,6 +17,7 @@ combine_as_imports=1
 
 [mypy]
 # Docs: https://mypy.readthedocs.io/en/latest/config_file.html
+# See #4853 to know why it's enabled by default.
```

Comment:
I would prefer to explain in the comment itself why it's enabled by default to keep things in one place.

Also ... as @prettyirrelevant said in discord perhaps there is a way to keep this on only for our code and off for all other stuff?

I guess(?) the way to do it is leave it `True` here which is the "generic" mypy section.

And then do a new section with it set to `False` that covers all of our code. Maybe like this

```
[mypy-rotkehlchen.*]
```

Reply:
Oh yeah that worked fine. Here is the output after defining the new section

![image](https://user-images.githubusercontent.com/73274/191019838-1e74949d-def3-43bd-a19d-63027047b809.png)